tsdb/chunkenc/value.go

```diff
@@ -16,23 +16,44 @@ package chunkenc
 import (
 	"bytes"
 	"encoding/binary"
+	"io"
+
+	"github.com/klauspost/compress/zstd"
 )
 
+// zstdFrameMagic is the magic beginning of all zstd compressed frames.
```

Comment:
is this a zstd standard? If so, can you link the spec?

Reply:
Yes, it's a zstd standard and copied from the compress zstd lib we're using.
https://github.com/klauspost/compress/blob/063ee1dad7a10b7caef0432d813ec3b8d72c8a8f/zstd/framedec.go#L59
https://www.rfc-editor.org/rfc/rfc8478.html#section-3.1.1

Reply:
I guess I'll add this to the source itself :)

Reply:
Thank you!